mesonbuild/build.py

```diff
@@ -1702,6 +1702,26 @@ def check_module_linking(self):
                                      'use shared_library() with `override_options: [\'b_lundef=false\']` instead.')
                     link_target.force_soname = True
 
+    def process_vs_module_defs_kw(self, kwargs: T.Dict[str, T.Any]) -> None:
+        if kwargs.get('vs_module_defs') is not None:
```

Comment:
Perhaps invert the logic to save an indentation and return early if no vs_module_defs kwarg exists?